_tests/integration/helper/git.go

```diff
@@ -0,0 +1,22 @@
+package helper
+
+import (
+	"fmt"
+	"github.com/bitrise-io/go-utils/command/git"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func GitClone(t *testing.T, dir, uri string) {
```

Comment:
nit: Now that we have an extracted git clone function that is used by every test, I'm wondering if there is anything we can do to speed up these slow test cases. Maybe executing `git clone --depth=1` in this function?

Reply:
That is a good one. I have tried to achieve this in the easiest way which does not need me to update the git package in the `go-utils`. The existing git functions are not flexible at all and I evaluated that I need to do a lot of things to achieve it. 

What I did was to manually inject the `--depth=1` option into the generated commands argument list before executing it. 🙈

Feel free to block it if you think it should be done in a proper way. 😄